saic-java-mqtt-gateway/src/main/java/net/heberling/ismart/mqtt/VehicleState.java

```diff
@@ -38,6 +38,7 @@ public class VehicleState {
   private Long refreshPeriodAfterShutdown;
   private RefreshMode refreshMode;
   private RefreshMode previousRefreshMode;
+  private int remoteTemperature;
```

Comment:
Should be Integer, to exactly detect an unset value. (even though 0 is probably unsupported by the cars anyway and if it would be supported it would be very odd) This makes it behave like the other configurable values in this class.

Reply:
0 in the car means off, however this is a "human" temperature in °C. Wonder how we should handle this for example for UK... :) In the state I would store real temperature values. Will test out in HA if it can somehow map to Farenheit, would be great to find someone from UK who could help us what temperature steps are shown there.

Reply:
The car also allows LO and HI, probably LO is "1" in the mapping (being 0==off and 2==16), and same would apply to the value after 28... Will test this out as soon as I can...